apps/condo/domains/acquiring/routes/paymentLinkRouter.js

```diff
@@ -119,6 +140,7 @@ class PaymentLinkRouter {
     }
 
     async handleRequest (req, res) {
+        this.context.req = req
```

Comment:
You have to remove it, since this.context are shared for all requests to the route

Reply:
Fixed. Pass only single header.